x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix.tsx

```diff
@@ -57,6 +58,24 @@ const TOGGLE_OFF = i18n.translate('xpack.ml.splom.toggleOff', {
 
 const sampleSizeOptions = [100, 1000, 10000].map((d) => ({ value: d, text: '' + d }));
 
+interface OptionLabelWithIconItipProps {
```

Comment:
nit: extra `I` in `OptionLabelWithIconItipProps`

Reply:
Thanks! Fixed in 247d514.